includes/discovery/sensors/current/adva.inc.php

```diff
@@ -0,0 +1,126 @@
+// ***** Sensors for ADVA FSP150EG-X Chassis
+// ******************************************
+
+if (starts_with($device['sysObjectID'], 'enterprises.2544.1.12.1.1')) {
+    // Define Sensors and Limits
+    $sensors = array
+                (
+                array(
+                        'sensor_name'     => 'psuOutputCurrent',
+                        'sensor_oid'      => '.1.3.6.1.4.1.2544.1.12.3.1.4.1.8',
+                        'multiplier'      => 1,
+                        'divisor'         => 1000,
+                        'low_limit'       => 1,
+                        'low_warn_limit'  => 1,
+                        'high_warn_limit' => 25,
+                        'high_limit'      => 30));
+    
+    foreach (array_keys($pre_cache['fsp150']) as $index1) {
+        foreach ($sensors as $entry) {
+            $sensor_name = $entry['sensor_name'];
+            if ($pre_cache['fsp150'][$index1][$sensor_name]) {
+                $multiplier      = $entry['multiplier'];
+                $divisor         = $entry['divisor'];
+                $low_limit       = $entry['low_limit'];
+                $low_warn_limit  = $entry['low_warn_limit'];
+                $high_warn_limit = $entry['high_warn_limit'];
+                $high_limit      = $entry['high_limit'];
+
+                $descr       = $pre_cache['fsp150'][$index1]['slotCardUnitName']." [#".$pre_cache['fsp150'][$index1]['slotIndex']."]";
+                $current     = $pre_cache['fsp150'][$index1][$entry];
+                $sensorType  = 'advafsp150';
+                $oid         = $entry['sensor_oid'].".".$index1;
+
+                discover_sensor(
+                    $valid['sensor'],
+                    'current',
+                    $device,
+                    $oid,
+                    $index1,
+                    $sensorType,
+                    $descr,
+                    $divisor,
+                    $multiplier,
+                    $low_limit,
+                    $low_warn_limit,
+                    $high_warn_limit,
+                    $high_limit,
+                    $current
+                );
+            }//End if sensor exists
+        }//End foreach $entry
+    }//End foreach $index
+    unset($sensors, $entry);
+}// ************** End of Sensors for ADVA FSP150CC Series **********
+
+// *************************************************************
+// ***** Sensors for ADVA FSP3000 R7
+// *************************************************************
+
+if (starts_with($device['sysObjectID'], 'enterprises.2544.1.11.1.1')) {
+    $multiplier = 1;
+    $divisor    = 1000;
+
+    if (is_array($pre_cache['fsp3kr7_Card'])) {
+        echo "psuEntry: ";
+
+        foreach (array_keys($pre_cache['fsp3kr7_Card']) as $index) {
+        //AC PSU Limits
+            if ($pre_cache['fsp3kr7_Card'][$index]['eqptPhysInstValuePsuAmpere']) {
+                if ($pre_cache['fsp3kr7_Card'][$index]['eqptPhysInstValuePsuAmpere'] > 1000) {
```

Comment:
Sorry, this was unfinished and forgotten business. Wanted to distinguish between AC/DC PSUs to set the correct limits. Since I kicked out the hardcoded limits, this went away as well.